paper-tabs.html

```diff
@@ -376,6 +378,28 @@
         }, 10);
       },
 
+      _isItem: function(node) {
+        return this.items.indexOf(node) >= 0;
+      },
+
+      _onIronItemsChanged: function(event) {
+        this._onTabSizingChanged();
+
+        var _bindActivateHandler = this._bindActivateHandler;
+
+        event.detail.addedNodes
```

Comment:
I would add a comment that this will be called after `this.items` is updated, so it will always be right.

A generic comment about what/why were doing this would also be nice. "Add a focus listener to each of the tabs, so that ..."

Same for removed?